src/java.base/windows/native/libjava/HostLocaleProviderAdapter_md.c

```diff
@@ -965,8 +965,9 @@ void getNumberPart(const jchar * langtag, const jint numberStyle, WCHAR * number
 void getFixPart(const jchar * langtag, const jint numberStyle, BOOL positive, BOOL prefix, WCHAR * ret) {
     DWORD pattern = 0;
     int style = numberStyle;
-    int got = 0;
+ // int got = 0;
 
+/*
```

Comment:
The warning received is that got is set but not used \- I had a look at getLocaleInfoWrapper and it seems like it has no side effects, its only purpose is to return a value, which is the variable got in this case. I'm pretty certain got not being used is a bug, and that it is needed somewhere, but someone forgot to hook it up to whatever needs it

Reply:
It does seem a bug, where the return value is not handled correctly. Can you please file a separate bug?

Reply:
> I had a look at getLocaleInfoWrapper and it seems like it has no side effects, its only purpose is to return a value

While `got` is not needed here, I am not convinced that `getLocaleInfoWrapper` is not used, so we shouldn't comment out all of the code, just remove `got`. `getLocaleInfoWrapper` updates the outbound variable `pattern` (see https://learn.microsoft.com/en-us/windows/win32/api/winnls/nf-winnls-getlocaleinfoex), which is used to get the right string pattern from `fixes`.

Usually, based on `got`, appropriate action is taken, but that's not needed in this case, because `pattern` is either updated or remains its fallback 0 value. So I presume that is why it is ignored in this case.

We could just add that `got` removal to this patch, what do you think @naotoj?

Reply:
Thanks, Justin. That is correct. Only `pattern` is relevant here. No need to check `got` so it can safely be removed.